dotCMS/src/integration-test/java/com/dotcms/publisher/bundle/business/BundleAPITest.java

```diff
@@ -0,0 +1,205 @@
+package com.dotcms.publisher.bundle.business;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.contenttype.exception.NotFoundInDbException;
+import com.dotcms.datagen.UserDataGen;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.business.DotPublisherException;
+import com.dotcms.publisher.business.PublishAuditHistory;
+import com.dotcms.publisher.business.PublishAuditStatus;
+import com.dotcms.publisher.business.PublishAuditStatus.Status;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.DateUtil;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Set;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BundleAPITest {
+
+    private static User adminUser;
+    private static BundleAPI bundleAPI;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)